cfme/infrastructure/pxe.py

```diff
@@ -742,6 +775,30 @@ def create(self, name, provision_type, cancel=False):
         main_view.flash.assert_success_message(msg)
         return system_image_type
 
+    def delete(self, cancel=False, *sys_objs):
+        """
+        This methods deletes the System Image Type using select option,
+         hence can be used for multiple delete.
+
+        Args:
+            cancel: This is the boolean argument required for handle_alert
+            sys_objs: It's System Image Types object
+        """
+        view = navigate_to(self, 'All')
+        rows = view.entities
```

Comment:
**REQUIRED:** Pretty sure entities has a search or find feature, this shouldn't be necessary